src/modules/Tab/Tab.js

```diff
@@ -64,7 +69,8 @@ class Tab extends Component {
   ]
 
   static defaultProps = {
-    menu: { attached: true, tabular: true },
+    menu: { attached: true, tabular: true, vertical: false, fluid: false },
+    grid: { segmentWidth: 12, tabWidth: 4 },
```

Comment:
In keeping with the Tab terminology, let's go with `paneWidth`  instead of `segmentWidth`.